bundles/binding/org.openhab.binding.gc100ir/src/main/java/org/openhab/binding/gc100ir/util/ItemUtility.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2010-2013, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.gc100ir.util;
+
+import java.util.HashMap;
+
+/**
+ * Utility class for storing and retrieving items.
+ * 
+ * @author Parikshit Thakur & Team
+ * @since 1.6.0
+ * 
+ */
+public class ItemUtility {
```

Comment:
please rename to something more meaningful since this class does not deal with Items but rather with GC100ItemBeans which in turn provide access to the item name but not (and that is perfectly fine) to the Item itself